src/templates/csr/fabric-video/index.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+	import type { GAMVariable } from '$lib/gam';
+	import Fabric from '$templates/components/Fabric.svelte';
+
+	export let Trackingpixel: GAMVariable;
+	export let Researchpixel: GAMVariable;
+	export let Viewabilitypixel: GAMVariable;
+
+	export let BackgroundScrollType: GAMVariable<'parallax' | 'none' | 'fixed'>;
+	export let BackgroundColour: GAMVariable;
+
+	export let BackgroundImage: GAMVariable;
+	export let BackgroundImagePosition: GAMVariable;
+	export let BackgroundImageRepeat: GAMVariable;
+	export let Layer1BackgroundImage: GAMVariable;
+	export let Layer1BackgroundPosition: GAMVariable;
+	export let Layer2BackgroundImage: GAMVariable;
+	export let Layer2BackgroundPosition: GAMVariable;
+	export let Layer3BackgroundImage: GAMVariable;
+	export let Layer3BackgroundPosition: GAMVariable;
+
+	export let MobileBackgroundImage: GAMVariable;
+	export let MobileBackgroundImagePosition: GAMVariable;
+	export let MobileBackgroundImageRepeat: GAMVariable;
+	export let MobileLayer1BackgroundImage: GAMVariable;
+	export let MobileLayer1BackgroundPosition: GAMVariable;
+	export let MobileLayer2BackgroundImage: GAMVariable;
+	export let MobileLayer2BackgroundPosition: GAMVariable;
+	export let MobileLayer3BackgroundImage: GAMVariable;
+	export let MobileLayer3BackgroundPosition: GAMVariable;
+
+	export let VideoURL: GAMVariable;
+	export let VideoBackupImage: GAMVariable;
+	export let MobileVideoBackupImage: GAMVariable;
+	export let VideoURLMobile: GAMVariable;
+	export let VideoAlignment: GAMVariable;
+
+	export let isXL = false;
```

Comment:
these don't need exported can be inline on the component below

Reply:
@Jakeii actually, this one can be removed totally as the Fabric component has a default value of true?

Reply:
exactly don't need to specify it, if it's the defaults